Introxt_Instagram.sh

```
@@ -46,37 +46,38 @@ echo "upto:"
+        # This resolves the 'nested brace in URL' error and '405 Not Allowed' error
+        curl -i "https://www.instagram.com/web/friendships/${user_id[$i]}/follow/" \
+             -H "X-Forwarded-For: ${ip_addr[$i]}" \
              -X POST \
-             # ...other headers...
              -o follow.json
         sleep 3s
     done
 done
-"""
\ No newline at end of file
+"""
```

Shellcheck (0.11.0)

[error] Couldn't parse this double quoted string. Fix to allow more checks.

(SC1073)